test/text/ttml_text_parser_unit.js

```diff
@@ -875,6 +883,43 @@ describe('TtmlTextParser', () => {
         {periodStart: 0, segmentStart: 0, segmentEnd: 0});
   });
 
+  it('parses cellResolution', () => {
+    verifyHelper(
+        [
+          {
+            startTime: 1,
+            endTime: 2,
+            cellResolution: [
+              Cue.cellResolution.COLUMNS,
+              Cue.cellResolution.ROWS,
+            ],
```

Comment:
If it doesn't matter what the cellResolution is in this output cue (since it's not using c units), you can omit this field.  verifyHelper below uses jasmine.objectContaining() to match, so any field you don't specify in the expectation won't count.

In fact, it seems that this test case doesn't have much to do with cellResolution at all, so maybe you could omit it.

Reply:
in fact this test should check that Cue has default cellResolution when ttp:cellResolution was not presented in ttml
I guess it would be nice to leave this check